cmd/server/reader.go

```diff
@@ -51,20 +51,18 @@ func fedWireDataFile(logger log.Logger) (io.Reader, error) {
 func attemptFileDownload(logger log.Logger, listName string) (io.Reader, error) {
 	routingNumber := os.Getenv("FRB_ROUTING_NUMBER")
 	downloadCode := os.Getenv("FRB_DOWNLOAD_CODE")
-
-	if routingNumber != "" && downloadCode != "" {
-		logger.Logf("download: attempting %s", listName)
-		client, err := download.NewClient(&download.ClientOpts{
-			RoutingNumber: routingNumber,
-			DownloadCode:  downloadCode,
-		})
-		if err != nil {
-			return nil, fmt.Errorf("client setup: %v", err)
-		}
-		return client.GetList(listName)
+	downloadURL := os.Getenv("FRB_DOWNLOAD_URL_TEMPLATE")
```

Comment:
I forgot to mention before but can we update the README and docs with this env variable?

Reply:
I've got some technical issues to figure out, but I'll circle back to this once that's done.